### PR TITLE
Onboarding improvements + request organization improvements

### PR DESCRIPTION
**Pull request by demenech**

### CODE REVIEW

File: frontend/src/components/dashboard/JoinOrganizationModal.tsx [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+    defaultValues: {
+      isCreation: false,
+      message: "",
+      confirmMembership: false,
+    },
+  });
+
+  const onboardMutation = api.user.onboard.useMutation({
+    // TODO: reset is not working
+    // TODO: toast is hiding too fast
```
Comment (Gutts-n) [inline frontend/src/components/dashboard/JoinOrganizationModal.tsx:108]: Were these TODOS done?

File: frontend/src/pages/auth/signup.tsx [hunk 2]
```diff
@@ -1,11 +1,6 @@
...
 import type { GetServerSideProps } from "next";
-import { getCsrfToken, signIn } from "next-auth/react";
-import { NextSeo } from "next-seo";
-import { useRouter } from "next/router";
-import { deleteCookie, setCookie } from "cookies-next";
-import { useEffect } from "react";
+import { getCsrfToken } from "next-auth/react";
+import LoginPage from "@components/auth/LoginPage";
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
```
Comment (Gutts-n) [inline frontend/src/pages/auth/signup.tsx:5]: You can remove this function as you created a wrapper for the entire page.

File: integration-tests/cypress.config.js
Comment (Gutts-n) [inline integration-tests/cypress.config.js:1]: is this file necessary?

File: integration-tests/cypress/fixtures/example.json
Comment (Gutts-n) [inline integration-tests/cypress/fixtures/example.json:1]: is this file necessary?

File: integration-tests/cypress/support/commands.js
Comment (Gutts-n) [inline integration-tests/cypress/support/commands.js:1]: is this file necessary?

File: integration-tests/cypress/support/e2e.js
Comment (Gutts-n) [inline integration-tests/cypress/support/e2e.js:1]: is this file necessary?

**Review by Gutts-n — CHANGES_REQUESTED** [review]

File: frontend/src/pages/auth/signup.tsx (hunk 2 above)
Comment (demenech) [reply frontend/src/pages/auth/signup.tsx:5]: I think it might be somehow necessary

File: frontend/src/components/dashboard/JoinOrganizationModal.tsx (hunk 1 above)
Comment (demenech) [reply frontend/src/components/dashboard/JoinOrganizationModal.tsx:108]: For future reference only, more like a FIXME
